third_party/terraform/tests/resource_bigquery_table_test.go

```diff
@@ -306,6 +329,58 @@ EOH
 `, datasetID, cryptoKeyName, tableID)
 }
 
+func testAccBigQueryTableRangePartitioning(datasetID, tableID string) string {
+	return fmt.Sprintf(`
+	resource "google_bigquery_dataset" "test" {
+		dataset_id = "%s"
+	}
+
+	resource "google_bigquery_table" "test" {
+		provider   = google-beta
+		table_id   = "%s"
+		dataset_id = "${google_bigquery_dataset.test.dataset_id}"
+
+		range_partitioning {
+			field = "id",
+			range {
+				start    = 1
+				end      = 10000
+				interval = 100
+			}
+		}
+
+		schema = <<EOH
+[
+	{
+		"name": "ts",
+		"type": "TIMESTAMP"
+	},
+	{
+		"name": "city",
+		"type": "RECORD",
+		"fields": [
+	{
```

Comment:
nit - indent

Reply:
Should I not match the indent from the previous function?

Reply:
oh, it's probably fine to be consistent then - mostly I was getting the feeling that these were at the same nesting as ts/city

Reply:
I agree it looks pretty weird in the file. Maybe both the indents can be cleaned up in a later commit or something